cloudstate/stateless_function_entity.py

```diff
@@ -0,0 +1,186 @@
+"""
+Licensed under the Apache License, Version 2.0.
+"""
+import inspect
+from dataclasses import dataclass, field
+from typing import Callable, List, MutableMapping
+
+from google.protobuf import descriptor as _descriptor
+
+from cloudstate.function_pb2 import _STATELESSFUNCTION
+from cloudstate.stateless_function_context import StatelessFunctionContext
+import logging
+
+
+@dataclass
+class StatelessFunction:
+    service_descriptor: _descriptor.ServiceDescriptor
+    file_descriptors: List[_descriptor.FileDescriptor]
+    unary_handlers: MutableMapping[str, Callable] = field(default_factory=dict)
+    stream_handlers: MutableMapping[str, Callable] = field(default_factory=dict)
+    stream_in_handlers: MutableMapping[str, Callable] = field(default_factory=dict)
+    stream_out_handlers: MutableMapping[str, Callable] = field(default_factory=dict)
+
+    @property
+    def persistence_id(self):
```

Comment:
I don't think we need a persistence_id because in the stateless case there will be no persistence involved

Reply:
this is the last issue I need to resolve: I haven't been able to get discovery to work without this.